app/models/pipeline_run.rb

```diff
@@ -33,12 +33,20 @@ class PipelineRun < ApplicationRecord
   STATUS_RUNNABLE = 'RUNNABLE'.freeze
   STATUS_ERROR = 'ERROR'.freeze # when aegea batch describe failed
   STATUS_LOADED = 'LOADED'.freeze
+  STATUS_EMPTY = 'EMPTY'.freeze
+  STATUS_LOADING = 'LOADING'.freeze
```

Comment:
Whose status?   Can we separate the status token namespaces for pipeline monitor vs result monitor.

Reply:
ok, I separated them in the code when I added a description. Some, such as STATUS_FAILED, are still shared between various entities. That can be the subject of another diff.